Lib/pty.py

```diff
@@ -170,3 +175,128 @@ def spawn(argv, master_read=_read, stdin_read=_read):
 
     os.close(master_fd)
     return os.waitpid(pid, 0)[1]
+
```

Comment:
Most manpages referenced are from Linux. However, they should be similar on the BSDs.